include/wx/univ/setup.h

```diff
@@ -1637,6 +1637,19 @@
 // Recommended setting: 1, required by wxMediaCtrl
 #define wxUSE_ACTIVEX 1
 
+// Enable WinRT support
+//
+// Default is 1 for compilers which support it, i.e. VS2012+ currently. If you
+// use an earlier MSVC version or another compiler and installed the necessary
+// SDK components manually, you need to change this setting.
+//
+// Recommended setting: 1
+#if defined(_MSC_VER) && _MSC_VER >= 1700 && !defined(_USING_V110_SDK71_)
+    #define wxUSE_WINRT 1
```

Comment:
OTOH I don't think it makes sense to enable this for wxUniv, this is used for native notifications that probably shouldn't be used by wxUniv.

Reply:
So, would adding it with a value of 0 to setup.h be correct?
Or editing the wx/msw/chkconf.h to set it to zero?

Tim S.

Reply:
Either would be fine.

Reply:
Okay, I have updated the pull request to set it to zero in setup.h and will be working on finding the linking error with core library. Laptop is getting hot so may have to stop for a few hours.

Tim S.